TinyKeychain/Keychain.swift

```diff
@@ -190,7 +190,7 @@ public struct Keychain {
      - returns: Either success, or a `Keychain.DeletionError`.
      */
     @discardableResult
-    public func deleteObject<O>(withKey key: Key<O>) -> KeychainResult<Void, DeletionError> {
+    public func deleteObject<O>(forKey key: Key<O>) -> KeychainResult<Void, DeletionError> {
```

Comment:
We could bike-shed on this. However, I think merging is the right idea, as more activity on this GitHub repo means more people using the framework. Also, more power and fame for you and me.

Reply:
Nothing better than for glory! Was going for consistency here. Either they are all ```with``` or all ```for```.